test/drivers/second_quantization/gaussiand/test_driver_gaussian_forces.py

```diff
@@ -133,6 +133,23 @@ def test_driver_jcf(self):
         result = driver.run()
         self._check_driver_result(self._get_expected_values(), result)
 
+    @unittest.skipIf(not _optionals.HAS_GAUSSIAN, "gaussian not available.")
+    def test_driver_xcf(self):
+        """Test the GaussianForcesDriver.from_molecule accept xcf argument"""
+        molecule = Molecule(
+            geometry=[
+                ("C", [-0.848629, 2.067624, 0.160992]),
+                ("O", [0.098816, 2.655801, -0.159738]),
+                ("O", [-1.796073, 1.479446, 0.481721]),
+            ],
+            multiplicity=1,
+            charge=0,
+        )
+
+        driver = GaussianForcesDriver.from_molecule(molecule, "6-31g", {}, "B3LYP")  # type: ignore
+        # test result has the value 'B3LYP'
+        self.assertIn("B3LYP", driver._jcf)
```

Comment:
If we keep this test simple like this, you should at least provide a value other than the default, otherwise this check does not really tell us whether the argument is being used correctly or not.